specification/ai/OpenAI.Assistants/examples/2024-02-15-preview/cancel_run.json

```diff
@@ -28,7 +28,7 @@
             "type": "code_interpreter"
           }
         ],
-        "incomplete_details": null,
+        "incomplete_details": "null",
```

Comment:
Yeah, this is incorrect. It should be `null` and it should pass because you have the Swagger properly annotated.